src/Pos/ReportServer/FileOps.hs

```diff
@@ -46,11 +65,9 @@ parseIndexEntry line = case T.splitOn "," line of
         ix <- mToE ("Couldn't read index: " <> a) $ readMaybe $ T.unpack a
         time <-
             mToE ("Couldn't parse utctime: " <> b) $
-            parseTimeM True defaultTimeLocale dateFormat (T.unpack b)
-        let fpath = T.unpack c
-        when ("/" `T.isInfixOf` c) $
-            Left $ "Filepath has '/' inside it: " <> c
```

Comment:
Why did we have this check earlier and don't have it anymore?

Reply:
Because now filepaths have `/` in them. Previously filepath (relative) was just `x`, now it's `date/type/x`.